src/Api/Api.php

```diff
@@ -680,14 +680,20 @@ public function responseFor404(): array
      */
     public function responseForException(Throwable $e): array
     {
+        if (isset($this->kirby) === true) {
+            $docRoot = $this->kirby->environment()->get('DOCUMENT_ROOT');
+        } else {
+            $docRoot = $_SERVER['DOCUMENT_ROOT'] ?? null;
+        }
```

Comment:
Doesn't your new `::getGlobally` method already do what these lines do? Is it returning different value in different instances?

````php
$docRoot = Environment::getGlobally('DOCUMENT_ROOT');
````

Reply:
Since we have an actual `$this->kirby` instance here, I think we should use it directly and not rely on the global instance. One could set up another `App` instance and create an `Api` instance from that.